flight/Libraries/math/misc_math.c

```diff
@@ -367,6 +367,27 @@ uint32_t randomize_int(uint32_t interval)
 	return rand_val % interval;
 }
 
+/**
+ * @brief Apply deadband to Roll/Pitch/Yaw channels
+ */
+void apply_channel_deadband(float *value, float deadband)
+{
+	if (deadband < 0.0001f) return; /* ignore tiny deadband value */
```

Comment:
New lines required for these two ifs.

Reply:
Just relocated this code... didn't change it.